diff --git a/vendor/github.com/alexellis/go-execute/pkg/v1/exec.go b/vendor/github.com/alexellis/go-execute/pkg/v1/exec.go
index a81d4d6..347e629 100644
--- a/vendor/github.com/alexellis/go-execute/pkg/v1/exec.go
+++ b/vendor/github.com/alexellis/go-execute/pkg/v1/exec.go
@@ -41,7 +41,7 @@ func (et ExecTask) Execute() (ExecResult, error) {
 	}
 
 	if et.PrintCommand {
-		fmt.Println("exec: ", et.Command, argsSt)
+		fmt.Fprintf(os.Stderr, "exec: %s %s\n", et.Command, argsSt)
 	}
 
 	var cmd *exec.Cmd
